presentation/src/main/java/com/android254/presentation/feed/view/FeedComponent.kt

```diff
@@ -112,7 +111,7 @@ fun FeedComponent(
             }
 
             ChaiBodyXSmall(
-                bodyText = feedPresentationModel.createdAt,
+                bodyText = getTimeDifference(feedPresentationModel.createdAt),
```

Comment:
This should happen during mapping so that we can test it. ideally, UI shouldn't have any logic

Reply:
great, will move it there.